src/main/java/com/palehorsestudios/alone/gui/controller/ItemSelectionWindowController.java

```diff
@@ -152,17 +151,20 @@ private void selectItems() {
                     .getKey()
                     .selectedProperty()
                     .addListener(
+                            // initItems.size() instead of count....
                             new ChangeListener<Boolean>() {
                                 public void changed(ObservableValue ov, Boolean old_val, Boolean new_val) {
-                                    if (count == 9) {
-                                        paneSelected.setDisable(true);
+                                    if (initItems.size() == 10) {
```

Comment:
having it >= 10 would make ensure it would work (if for whatever reason it went beyond 10). Shouldn't be an issue but would recommend it